main/settings.py

```diff
@@ -732,6 +768,12 @@
     },
 }
 
+if ENABLE_WAYBACK_TASKS:
+    CELERY_BEAT_SCHEDULE["update-wayback-jobs-status"] = {
+        "task": "external_resources.tasks.update_wayback_jobs_status_batch",
+        "schedule": UPDATE_WAYBACK_JOBS_STATUS_FREQUENCY,
+    }
+
 if CHECK_EXTERNAL_RESOURCE_TASK_ENABLE:
```

Comment:
This needs to be dealt with as well, but probably i a separate task along with changes required to convert it to a posthog feature flag